tensilelite/Tensile/Components/GSU.py

```diff
@@ -409,12 +412,16 @@ def noLoadLoop(self, writer, kernel, tensorParametersA, tensorParametersB, pack)
             # (2. OptNLL : No PAP global-read (For PAP=OFF, or PAP=ON but the last tile))
             #  -> this is unified with 1. global-read is invalidated at the last tile.
             # 3. OrdinaryNLL (Not Opt.)
-            writer.saveLocalPointers(kernel, tensorParametersA, tensorParametersB)
-            # deepCopy packCode for OptNLL noLoadLoop
-            deepCopyPack = fastdeepcopy(pack)
-            noLoadLoopModules = writer.noLoadLoop(kernel, tensorParametersA, tensorParametersB, isOptNLL=True, isNGLL=False, pack=deepCopyPack)
+
+            noLoadLoopModules = Module("noLoadLoop")
+            for NLLindex in range(1, NLLnum + 1):
```

Comment:
It is more common to use 0 and 1 for NLLindex. Suggest you modifying it along with DTVA future changes

Reply:
NLLindex expresses the unroll loop index asm code.
It is 1 and 2 even before my change.

`/******************************************/`
`/* Unrolled Loop 1/2 - Begin              */`
`/******************************************/`
...
`/******************************************/`
`/* Unrolled Loop 2/2 - Begin              */`
`/******************************************/`
'''

Reply:
The loop index is 0 and 1 for unrolled loop. It only plus 1 when printing.

> module.addComment2("Unrolled**_ Loop %u/%u - Begin" % (**lc+1**, loopCopies))

Reply:
I think I put wrong comment. NLL is not unroll loop.
NLL = NoLoadLoop.
The intention here is as I explained.
Essentially, it does not make difference.
I can make the change as you suggested.

Reply:
Updated